### PR TITLE
Add `Community Mentorship Program Summer 2023: Signups Now Open` news post 

### PR DESCRIPTION
**Pull request by osuWhite**
Will probably need to update the forum thread link shortly before launch.

### CODE REVIEW

File: news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Community Mentorship Program Summer 2023 Signups Now Open"
```
Comment (0x84f) [inline news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md:3]: I think this looks better
```suggestion
title: "Community Mentorship Program Summer 2023: Signups Now Open"
```

File: news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, storyboarders, and even prospective mentors to develop their skills under the guidance of experienced members of the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1716981), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```
Comment (0x84f) [inline news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md:15]: this is subjective
```suggestion
If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1716981), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you to apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```

**Review by 0x84f — COMMENTED** [review]
some subjective stuff

**Comment by osuWhite** [comment]
@0x84f can you check that your 2nd suggestion got applied properly? I hit "resolve" but that didn't seem to actually apply it LOL

**Comment by 0x84f** [comment]
Looks like you sorted it out

**Comment by 0x84f** [comment]
> Ok idk what the merge branch thing does but hopefully that doesn't mess anything up LOL. Should be good to go now though.

it just updates your "c24" branch with whatever differences there are compared to the master branch, no need to do that as the branch gets merged with the latest version of master before going live

File: news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, storyboarders, and even prospective mentors to develop their skills under the guidance of experienced members of the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1772532), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you to apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```
Comment (Walavouchey) [inline news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md:15]:
```suggestion
If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1772532), you'll be able to apply right away. If you're not sure whether you'll make the cut, we encourage you to apply anyway! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```

**Review by Walavouchey — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**